README.md

```diff
@@ -7,19 +7,15 @@ Features:
 * Integrate [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/) notebooks to the Open edX courseware.
 * Fully editable notebooks and student workspaces.
 * Simple integration of notebooks from public git repositories.
+* Works with OpenEdx Oreo, Palm, Quince
```

Comment:
Oreo would have been a lovely name, but it was Olive :) Let's not add this line, it's unnecessary.

Reply:
Sorry. I confused Android releases with OpenEdx :)

Reply:
I have updated this.